changelog/fix-9651-record-tracks-event-capital-view-offer-redirect

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
+Comment: Insignificant user-facing change – adding a new tracks event when the merchant is redirected to a Stripe Capital offer
+
```

Comment:
IMO it's OK to add a changelog for this change. We're not hiding it from the user.

```suggestion

When tracking is enabled, send a tracks event when the merchant is redirected to a Stripe Capital offer.
```